client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -140,19 +141,42 @@ class AdsFormSettings extends Component {
 		return <Card href={ linkHref }>{ translate( 'Manage ad placements' ) }</Card>;
 	}
 
+	wpcomPlacementControls() {
+		const { isLoading, translate } = this.props;
+		const isWordAds = this.props.site.options.wordads;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Ad Placements' ) }>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Additional ad placements' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/wordads-settings-single-page)